hugo/layouts/shortcodes/community-bio.html

```diff
@@ -0,0 +1,47 @@
+<style> .reading-time { display:none !important; }</style>
+<div class="bio">
+    <img src='{{"images/gardener-logo-light.svg" | relURL}}' alt="Logo" class="logo">
+
+    <h3>Gardener</h3>
+    <a>Homogeneous Kubernetes clusters at scale on any infrastructure.</a>
+        <div class="social">
+            <a target="_blank" href="https://www.youtube.com/channel/@GardenerProject">
+                <img src='{{"images/community/youtube-logo-green.svg" | relURL}}' class="bio-icon" />
+            </a>
+            <a target="_blank" href="https://x.com/GardenerProject">
+                <img src='{{"images/community/x-logo-green.svg" | relURL}}' class="bio-icon" />
+            </a>
+        </div>
+        <div class="blocks">
+            <a href="/" target="_blank">
+                <div class="block">
+                    <img src='{{"images/gardener-logo.svg" | relURL}}' class="bio-icon" alt="GitHub Logo" />
+                    Home page
+                </div>
+            </a>
+            <a href="https://github.com/gardener" target="_blank">
+                <div class="block">
+                    <img src='{{"images/community/github-mark-logo-green.svg" | relURL}}' class="bio-icon" alt="GitHub Logo" />
+                    Github
+                </div>
+            </a>
+            <a href="https://gardener-cloud.slack.com" target="_blank">
+                <div class="block">
+                    <img src='{{"images/community/slack-logo-green.svg" | relURL}}' class="bio-icon" alt="GitHub Logo" />
```

Comment:
Instead of using a generic alt we should make it specific
```suggestion
                    <img src='{{"images/community/slack-logo-green.svg" | relURL}}' class="bio-icon" alt="Slack Logo" />
```

Reply:
A copy-paste mistake, will edit in the PR that addresses the mobile look, thanks for noticing.